lua/entities/gmod_wire_expression2/core/cl_http_cl.lua

```diff
@@ -0,0 +1,131 @@
+--[[
+
+	Clientside HTTP library
+	 - swadical 🍌
```

Comment:
There's already lots of parts of the code that have author attribution for people who are no longer part of the project. I know how it feels to be proud of writing a significant chunk of code, but the info will always be in the Git history and author attributions usually just serve to make the situation more confusing.